integration/setup/setup.go

```diff
@@ -69,5 +72,19 @@ func installResources(ctx context.Context, config Config) error {
 		}
 	}
 
+	// Ensure Chart CRD creation
```

Comment:
This makes sense to me as we need the chart CRD.

Later when we migrate the chartoperator resource from cluster-operator to app-operator we can remove this.